src/thorlabs_cube/driver/tcube/tpz.py

```diff
@@ -153,10 +152,10 @@ async def set_input_volts_source(self, volt_src):
             The values can be bitwise or'ed to sum the software source with
             either or both of the other source options.
         """
-        payload = st.pack("<HH", 1, volt_src)
+        payload = st.pack("<HH", _CHANNEL, volt_src)
```

Comment:
```suggestion
        payload = st.pack("<HH", Tpz._CHANNEL, volt_src)
```
Use class constants.